src/rvsdg/egglog_optimizer/interval-analysis.egg

```diff
@@ -65,3 +76,74 @@
         (union gamma (OperandGroup (SubstVecOperandAll (vec-get outputs 0) inputs)))
       )
 )
+
+; HACK: Table of fucking integers
```

Comment:
I would also put a TODO 
 @rtjoa I think you are working on table for ranges of integers?

Reply:
I was not, but I just talked to Anjali and it seems like `VecOperand-get` is sufficient for this PR.

The range rules would be straightforward to write, but I'm unsure if we need them in general.